src/app/components/ContactFieldProperty.js

```diff
@@ -32,19 +32,10 @@ const ContactFieldProperty = ({ field, value, uid, onChange, ...rest }) => {
     }
 
     if (field === 'bday' || field === 'anniversary') {
-        const m = moment(value);
-        if (value === '' || m.isValid()) {
+        const date = value === '' ? toDate(Date.now()) : parseISO(value);
```

Comment:
What do you think about:
```suggestion
        const date = value === '' ? new Date() : parseISO(value);
```

Reply:
Yes have you considered using `new Date()` instead of `toDate`, I believe they function pretty much the same?

Reply:
Yeah, that's much better. It's what happens when you code before going to bed :sweat_smile: